include/cpuinfo.h

```diff
@@ -255,6 +259,9 @@ enum cpuinfo_vendor {
 	 * Sold its ARM designs in 1997. The last processor design was released in 1997.
 	 */
 	cpuinfo_vendor_dec       = 57,
+
+	/** Loongson Technologies, Inc. Vendor of LOONGARCH processor microarchitectures. */
+	cpuinfo_vendor_loongarch     = 258,
```

Comment:
You're putting the new value in the "defunct CPU vendors" section, are you shutting your production yet? If not, that's a mistake, and you should be inserting the definition in the section above, not here.

Also this list is representing companies, so this should be "loongson" instead of "loongarch", BTW the "LOONGARCH" should really be "LoongArch".